tests/scripts/all.sh

```diff
@@ -832,9 +832,21 @@ component_test_no_use_psa_crypto_full_cmake_asan() {
     if_build_succeeded env OPENSSL_CMD="$OPENSSL_NEXT" tests/compat.sh -e '^$' -f 'ARIA\|CHACHA'
 }
 
+component_test_check_params_tests () {
```

Comment:
Suggest a more descriptive name, like `component_test_check_mbedtls_param_failed()`. It's a bit verbose if you also add "tests" into the name to describe that this only runs the tests. I think it more important to emphasize that it is testing the `mbedtls_param_failed()` function, more so than that this runs only the tests.

Reply:
It's the only component that runs the tests that verify that calls to API functions that pass invalid parameters result in a call to `MBEDTLS_PARAM_FAILED`. I renamed it to `test_check_params_functionality`.